pages/signup-review.js

```diff
@@ -0,0 +1,492 @@
+      return `${
+        formFields.option.yes.charAt(0).toUpperCase() +
+        formFields.option.yes.slice(1).toLowerCase()
+      }, ${disabilityDetails}`;
+    }
+  };
+
+  const incomeString = (income) => {
+    if (!income) {
+      return "NA";
+    } else if (income === "preferNotToAnswer") {
+      return formFields.option.preferNotAnswer;
+    } else {
+      switch (income) {
+        case "30kLess":
+          return formFields.option.income1;
+        case "30kto60k":
+          return formFields.option.income2;
+        case "60kto100k":
+          return formFields.option.income3;
+        case "100kto150k":
+          return formFields.option.income4;
+        case "150kMore":
+          return formFields.option.income5;
+        default:
+          break;
+      }
+    }
+  };
+
+  const indigenousString = (nativeStatus) => {
+    if (!nativeStatus || nativeStatus === "N/A") {
+      return "NA";
+    } else if (nativeStatus === "preferNotToAnswer") {
+      return formFields.option.preferNotAnswer;
+    } else {
+      switch (nativeStatus) {
+        case "firstNations":
+          return formFields.option.firstNations;
+        case "métis":
+          return formFields.option.metis;
+        case "inuit":
+          return formFields.option.inuk;
+        default:
+          break;
+      }
+    }
+  };
+
+  const publicServantString = (publicServant) => {
+    if (!publicServant) {
+      return "NA";
+    } else {
+      return (
+        formFields.option[publicServant].charAt(0).toUpperCase() +
+        formFields.option[publicServant].slice(1).toLowerCase()
+      );
+    }
+  };
+
+  const handleSubmit = async (e) => {
+    // submit data to the api and then redirect to the thank you page
+    const response = await fetch("/api/sign-up", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(formData),
+    });
+
+    // if the response is good, redirect to the thankyou page
+    if (response.status === 201 || response.status === 200) {
+      sessionStorage.setItem("email", formData.email);
+      // Remove form data from session storage since it's no longer needed
+      delete formData["confirmEmail"];
+      sessionStorage.removeItem("formData");
+      let maskedEmail = maskEmail(formData.email);
+
+      await router.push({
+        pathname: "/thankyou",
+        query: { e: maskedEmail, ref: "signup" },
+      });
+    } else if (response.status === 400 || 500) {
+      await router.push("/500");
+    }
+  };
+
+  useEffect(() => {
+    if (props.adobeAnalyticsUrl) {
+      window.adobeDataLayer = window.adobeDataLayer || [];
+      window.adobeDataLayer.push({ event: "pageLoad" });
+    }
+    setFormData(JSON.parse(sessionStorage.getItem("formData")));
+  }, []);
+
+  return (
+    <>
+      <Layout
+        locale={props.locale}
+        langUrl={
+          props.locale === "en" ? pageData.scPageNameFr : pageData.scPageNameEn
```

Comment:
Good call on this, the rest of our pages are still using values from `common.json` but I like your way better